### PR TITLE
Life procs duplicate fix

### LINKED ISSUE #4824: Monkeys don't see white screen when flashbanged
As monkey, being flashbanged doesn't show the white flash. It does everything else though.
Also, when unconscious and dying, the white screen overlay doesn't appear.

### LINKED ISSUE #7815: Welding helmets no longer obscure the vision
Most likely culprit is #7770, but im not 100% sure on this
Someone fix this please

### PR DESCRIPTION
**Pull request by phil235**
This removes a lot of duplicated code in life and its subprocs for carbon mobs.
fixes #4824
fixes #7815 

Added a 7th alien health icon (so that there's the same number of icon for all carbon mobs with health icons)
Fixes cyborg and monkey not seeing any flash when flashed or flashbanged.

There's some other changes: aliens being able to be dizzy, jittery or hallucinate. I tried to adjust the oxyloss when lacking air or in crit (the values were different for humans and monkeys). Brain mob can be unconscious now. Probably more small adjustments.

Most of life subprocs for humans is still handled in species and its datums.

Thanks to @RemieRichards , I stole most of her code :monkey_face:

### CODE REVIEW

**Comment by Miauw62** [comment]
The son of God has risen again, judgement is near!

All of my :+1: for this PR
Op 21-feb.-2015 00:47 schreef "phil235" <notifications@github.com>:

**Comment by Iamgoofball** [comment]
@Miauw62 cyberspace when

**Comment by RemieRichards** [comment]
@Iamgoofball in the 8 seconds it takes us to notice the sun has burned out.

File: code/modules/mob/living/carbon/slime/death.dm [hunk 1]
```diff
@@ -24,4 +24,7 @@
 	if(ticker && ticker.mode)
 		ticker.mode.check_win()
 
-	return ..(gibbed)
\ No newline at end of file
+	return ..(gibbed)
+
+/mob/living/carbon/slime/gib()
```
Comment (Cheridan) [inline code/modules/mob/living/carbon/slime/death.dm]: I thought qdel wasn't safe to use with mobs?

Comment (phil235) [inline code/modules/mob/living/carbon/slime/death.dm]: Woops, yeah it needs a death(1) before qdelling.

**Pull request merged**